lib/src/chrome.dart

```diff
@@ -14,21 +14,19 @@ const _linuxExecutable = 'google-chrome';
 const _macOSExecutable =
     '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
 const _windowsExecutable = r'Google\Chrome\Application\chrome.exe';
-var _windowsPrefixes = [
-  Platform.environment['LOCALAPPDATA'],
-  Platform.environment['PROGRAMFILES'],
-  Platform.environment['PROGRAMFILES(X86)']
-];
+const _windowsPrefixes = ['LOCALAPPDATA', 'PROGRAMFILES', 'PROGRAMFILES(X86)'];
```

Comment:
If you're willing to set the lower SDK constraint to `2.2` (which I think you did) you could use a `Set` literal here!